src/server/pps/server/worker_rc.go

```diff
@@ -181,6 +190,20 @@ func (a *apiServer) getWorkerOptions(rcName string, parallelism int32, transform
 		Name:      "pach-bin",
 		MountPath: "/pach-bin",
 	})
+
+	volumes = append(volumes, api.Volume{
+		Name: client.PPSHostPathVolume,
+		VolumeSource: api.VolumeSource{
+			HostPath: &api.HostPathVolumeSource{
+				Path: client.PPSHostPath,
```

Comment:
How come PPSHostPath is in client? (vs. server, since it seems like we want to hide it from clients)